starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="utf-8">
   <title>Apple Pie Recipe</title>
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="./css/main.css">
+
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <div class='Menu'>
+  <div class="top" id="Apple">
+      <h1>Apple Pie</h1>
+  </div>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+  <div id='Container' >
+    <p class="Content">This was my grandmother's apple pie recipe. I have never seen another one
+    quite like it. It will always be my favorite and has won me several first place
+    prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+    
+    <img class="info" src="./images/recipe-info.png" alt="recipe-attributes"/>
+    
+    <p class="horizontal-line Content"></p>
+    <h3 class="Content">Ingredients</h3>
+    <ul class="Content">
```

Comment:
You can style lists too :) Some hints about list styling here:

https://css-tricks.com/almanac/properties/l/list-style/